### ISSUE TITLE
Reload Plugins not working

### ISSUE THREAD

**Issue by raguay** [fman-community/fman#482]
Every time I reload the plugins, I get this error:

```
Command 'ReloadPlugins' raised error.

Traceback (most recent call last):
  File "fman/impl/plugins/command_registry.py", line 52, in _execute_command
  File "/Applications/fman.app/Contents/Resources/Plugins/Core/core/commands/__init__.py", line 1665, in __call__
    unload_plugin(plugin)
  File "fman/__init__.py", line 268, in unload_plugin
  File "fman/impl/plugins/__init__.py", line 35, in unload_plugin
  File "fman/impl/plugins/plugin.py", line 216, in unload
  File "fman/impl/plugins/plugin.py", line 50, in _unregister_directory_pane_command
  File "fman/impl/plugins/command_registry.py", line 70, in unregister_command
KeyError: 'open_if_directory'
```

**Comment by mherrmann** [comment]
This was a bug in [fman 1.1.0](https://fman.io/changelog#v1.1.0). Which version of fman are you on? (I'm almost 100% pretty certain you know that you can see the version with the `About` command.)

**Comment by raguay** [comment]
I'm using 1.1.8. You might have a regression bug then.

**Comment by raguay** [comment]
I also thought this was something that had been fixed.

**Comment by mherrmann** [comment]
I think I know what the problem was. It's a different (though related) bug than the one in 1.1.0. I'll release the fix with v1.1.9.

**Comment by mherrmann** [comment]
[fman 1.1.9](https://fman.io/changelog#v1.1.9) is live. I believe it fixes the bug.
